### ISSUE TITLE
[2019] Multi-passage BERT: A Globally Normalized BERT Model for Open-domain Question Answering

### ISSUE THREAD

**Issue by cfiken** [cfiken/paper-reading#35]
### タイトル
Multi-passage BERT: A Globally Normalized BERT Model for Open-domain Question Answering
### リンク
https://arxiv.org/abs/1908.08167
### 著者
Zhiguo Wang, Patrick Ng, Xiaofei Ma, Ramesh Nallapati, Bing Xiang
### 投稿日付
EMNLP 2019
## 1. どんなもの？

OpenDomain な QA のタスクは BERT の登場で大きく改善されたが、既存手法は一つの質問に対して複数の候補文をそれぞれ独立に質問-候補文ペアとして学習し、それぞれでスコアの高い回答を選択している。
本研究では、既存手法では候補となる文章全体でのスコアが適切に計算できないなどの理由から、全てで Normalize しスコアを計算するなどの工夫を加えた Multi-passage BERT を提案。
OpenSQuAD, TriviaQA, Quasar-T, SearchQA で全ての SoTA を更新した。

## 2. 先行研究と比べてどこがすごい？

全ての候補文章から回答を予測する global normalization がよりモデルを安定化させることや、その他 ranker の貢献や候補文の stride などの効果をいくつかの実験により確認した。

## 3. 技術や手法のキモはどこ？

通常の BERT による QA タスクでは、質問 + 候補文ペアをモデルに入力し、回答の開始と終了位置を予測する。候補文ペアが複数ある場合、それぞれ独立にこの処理を行い、最もスコアの高いものを選ぶ。
Multi-passage BERT では、全ての候補文について logits を計算したあとに、全ての候補文で Softmax を取ることで、全体でのスコアを計算する。
他にも、各候補文の ranker も同様に全体でスコアを作成する、stride を導入するなどしてモデルを改善できた。

## 4. どうやって有効だと検証した？

OpenSQuAD, TriviaQA, Quasar-T, SearchQA の4つのデータセットで検証。
全てのデータセットで SoTA を更新した。
![スクリーンショット 2019-12-25 01 33 06](https://user-images.githubusercontent.com/1255485/71420078-837afc00-26b6-11ea-9606-fb79c19add7b.png)

## 5. 議論はある？

今後 passage 間の相関を取り入れたいとのこと.

## 6. 次に読むべき論文は？

- Simple and Effective Multi-Paragraph Reading Comprehension
    - https://arxiv.org/abs/1710.10723
    - Christopher Clark, Matt Gardner, ACL 2018
    - multi paragraph な QA に関する研究、 global normalizing のところで引用されていた
